packages/aws-cdk-lib/aws-dynamodb/lib/table-v2.ts

```diff
@@ -636,6 +685,11 @@ export class TableV2 extends TableBaseV2 {
         ? props.readCapacity._renderReadCapacity()
         : this.readProvisioning,
       tags: props.tags,
+      readOnDemandThroughputSettings: props.maxReadRequestUnits
+        ? { maxReadRequestUnits: props.maxReadRequestUnits }
```

Comment:
why it can't be the same as `writeOnDemandThroughputSettings: this.maxWriteRequestUnits
        ? { maxWriteRequestUnits: this.maxWriteRequestUnits }
        : undefined` ?

Reply:
This is because writes limits must be synced across all replicas. Reads do not follow this rule and can be defined individually per table replica.